### PR TITLE
Masq 76 common integration

### PR DESCRIPTION
**Pull request by leventdem**

### CODE REVIEW

File: src/library/masq.js [hunk 1]
```diff
@@ -5,8 +5,10 @@ import swarm from 'webrtc-swarm'
 import pump from 'pump'
 import uuidv4 from 'uuid/v4'
 import { promisifyAll } from 'bluebird'
+import common from '../../node_modules/masq-common/dist/index'
```
Comment (G-Ray) [inline src/library/masq.js]: can you do this instead ? `import common from masq-common/dist/index'`

File: src/library/masq.js [hunk 2]
```diff
@@ -174,14 +176,14 @@ class Masq {
 
       const sendAuthorized = async (peer, userAppDbId) => {
         const data = { msg: 'authorized', userAppDbId }
-        const msg = await encryptMessage(this.key, data)
-        peer.send(msg)
+        let encryptedMsg = await encrypt(this.key, data, 'base64')
```
Comment (G-Ray) [inline src/library/masq.js]: please use `const` as before

File: src/library/masq.js (hunk 1 above)
Comment (leventdem) [reply src/library/masq.js]: yes, done.

File: src/library/masq.test.js [hunk 3]
```diff
@@ -274,16 +271,16 @@ describe('masq protocol', async () => {
...
 
         // sw.close()
-        const encData = await encryptMessage(cryptoKey, {
+        let message = {
           msg: 'connectionEstablished'
-        })
-
-        peer.send(encData)
+        }
+        let encryptedMsg = await encrypt(cryptoKey, message, 'base64')
```
Comment (G-Ray) [inline src/library/masq.test.js]: always prefer `const`

Comment (leventdem) [reply src/library/masq.test.js]: ok

File: src/library/masq.js (hunk 2 above)
Comment (leventdem) [reply src/library/masq.js]: ok

**Comment by G-Ray** [comment]
Can you remove the functions which have been moved to the new module from `src/library/utils.js` please ?

EDIT: OK fine it has been deleted

**Review by G-Ray — APPROVED** [review]

**Review by G-Ray — APPROVED** [review]

**Pull request merged**